docs/docs/types/message.md

```diff
@@ -0,0 +1,48 @@
+Focused on deliverying outbound communication. Perhaps announcing a product launch, new content, top-level alert, etc. Message modules are simple text modules which can be used in a variety of formats.
+
+``` javascript
+var module = pathfora.Message({
+	// module settings
+});
+
+pathfora.initializeWidgets([ module ]);
+```
```

Comment:
I think it would be valuable to include an actual example set of module settings instead of the comment. The reason being docs are often used just to recall names. If someone knows they can customize the message but they forget the name of the key in the config is `msg`, having the answer in the first code example answers their question immediately.